<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatWindow</class>
 <widget class="QDialog" name="ChatWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btnSend">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>290</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtMessage">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="txtDisplayMessages">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>221</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
